src/openrct2-ui/input/MouseInput.cpp

```diff
@@ -1341,9 +1400,7 @@ void InputStateWidgetPressed(
                                 dropdown_index = gDropdownDefaultIndex;
                             }
                         }
-                        WindowEventDropdownCall(
-                            cursor_w, cursor_widgetIndex,
-                            (gDropdownIsColour) ? ColourToPaletteIndex(dropdown_index) : dropdown_index);
+                        WindowEventDropdownCall(cursor_w, cursor_widgetIndex, dropdown_index);
```

Comment:
I've changed it so that it is a true dropdown index that is always passed to the dropdown event. This means all users of the colour dropdown now need to convert the ddix into a colour.

Reply:
No they have always only ever had the colour never the ddidx.